### PR TITLE
Requester portfolio design update

### PR DESCRIPTION
**Pull request by Macbull**
Updated design of requester portfolio (with more relevant information): 

requester-portfolio6

### CODE REVIEW

File: staticfiles/templates/requester/portfolio.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
+[
```
Comment (nistala) [inline staticfiles/templates/requester/portfolio.json]: this is a fixture, can we move this to the backend to the appropriate folder?

**Comment by nistala** [comment]
one comment, otherwise LGTM

**Comment by Macbull** [comment]
@nistala  : moved the file to fixtures

**Comment by neilthemathguy** [comment]
Merged. Collaboration reference #255

**Pull request closed without merging**